README.md

```diff
@@ -1,296 +1,93 @@
+    - `├──`[`zowe-explorer`](./packages/zowe-explorer) — core Zowe Explorer source files, ReadMe, Changelog, and more.
 
-![Save](/docs/images/ZE-safe-save.gif?raw=true "Save")
-<br /><br />
+## Build Locally
 
-### Create a new PDS and a PDS member
+Ensure that you have met the [software requirements](#requirements) before you can build and test your Zowe Explorer.
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Click the **Create New Data Set** button to create a PDS.
-4. From the drop-down menu, select the type of PDS that you want to create.
-5. Enter a name for the PDS.
-   The PDS is created.
-6. To create a member, right-click the PDS and select **Create New Member**.
-7. Enter a name for the member.
-   The member is created.
+Clone the repository and run `yarn workspace vscode-extension-for-zowe package` to build a VSIX file and start working with the extension.
 
-![Create](/docs/images/ZE-cre-pds-member.gif?raw=true "Create")
-<br /><br />
+1. Clone the repository by issuing the following command:
 
-### Delete a PDS member and PDS
+   ```shell
+   git clone --origin=upstream --branch=main --single-branch https://github.com/zowe/vscode-extension-for-zowe.git
+   ```
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open the profile and PDS containing the member.
-4. Right-click on the PDS member that you want to delete and select **Delete Member**.
-5. Confirm the deletion by clicking **Yes** on the drop-down menu.
+2. From your local copy of the repository, issue the following commands:
 
-   **Note:** Alternatively, you can select 'No' to cancel the deletion.
+   ```shell
+   yarn install
+   yarn workspace vscode-extension-for-zowe package
+   ```
 
-6. To delete a PDS, right-click the PDS and click **Delete PDS**, then confirm the deletion.
+After you create a VSIX file, install the extension to VS Code.
 
-   **Note:** You can delete a PDS before you delete its members.
+1. Navigate to the Extensions menu in VS Code and click the `...` button in the top-left corner of the pane.
+2. Select Install from VSIX and select the .vsix file that was created by your `yarn workspace vscode-extension-for-zowe package` command.
+3. Restart Visual Studio Code.
 
-![Delete](/docs/images/ZE-del-pds-member.gif?raw=true "Delete")
-<br /><br />
+You can now use the extension.
 
-### View and access multiple profiles simultaneously
+For more information on how to run tests for the extension, see [Developer's ReadMe](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/Developer's%20ReadMe.md).
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Click the **Add Profile** button on the right of the **DATA SET** explorer bar.
-4. Select the profile that you want to add to the view as illustrated by the following screen.
+## Available Documentation
```

Comment:
Since we say we are linking to documentation here, it might be helpful to link directly to the readme part (i.e. anchor tag) of these repo package pages. For example:
- For the Zowe Explorer readme: https://github.com/zowe/vscode-extension-for-zowe/tree/master/packages/zowe-explorer#zowe-explorer
- For the extensibility API readme: https://github.com/zowe/vscode-extension-for-zowe/tree/master/packages/zowe-explorer-api#extensibility-api-for-zowe-explorer

And so on.